frontend/src/Editor/Inspector/EventManager.jsx

```diff
@@ -343,7 +343,8 @@ export const EventManager = ({
                 <CodeHinter
                   theme={darkMode ? 'monokai' : 'default'}
                   currentState={currentState}
-                  onChange={(value) => handlerChanged(index, 'contentToCopy', value)}
+                  initialValue={event.text}
+                  onChange={(value) => handlerChanged(index, 'text', value)}
```

Comment:
```suggestion
                  initialValue={event.contentToCopy}
                  onChange={(value) => handlerChanged(index, 'contentToCopy', value)}
```

Reply:
@kavinvenkatachalam I have made the changes...so it should work now ...kindly review!